mohitahlawat2001/Loco-Musica
# Implemented Google authentication using Firebase

**Description:** 
- Added Google Sign-In functionality using Firebase Authentication.
-  Users can log in via their Google account, and their details are stored in Firebase.
- Attached are screenshots showing the Google sign-in popup.
- Used the react-google-button library for the Google Sign-In button.

**React Dependency:**

`npm install react-google-button`

**Firebase Setup Instructions:**
To test this feature, follow these steps:

- Go to your Firebase Console and create a new project (if not already created).
- Navigate to **Authentication** > Sign-in method and enable Google.
- Add the **Authorized domains** (make sure` localhost` is included for local testing).
- Replace the Firebase configuration details in the project’s `.env` file with your Firebase project's API keys.

**Screenshots:**

![20241005_155940](https://github.com/user-attachments/assets/1f50f371-6fcb-4c26-a276-7ee857f6a7e1)
![Screenshot_20241005-160408_Samsung Notes](https://github.com/user-attachments/assets/405eb398-c1c8-444c-974f-d6518d02893e)
